### PR TITLE
Swaps from nested namespaces

### PR DESCRIPTION
**Pull request by KipHamiltons**
All this PR does is change to use one namespace scope in each C++ file with nested namespaces, then reformats the files.

### CODE REVIEW

**Comment by TrentHouliston** [comment]
For those reviewing this, I highly recommend you turn on the option to ignore white space changes!

File: shared/utility/math/geometry/UnitQuaternion.hpp [hunk 1]
```diff
@@ -24,128 +24,125 @@
...
+            inline arma::subview_col<double> imaginary() {
+                return rows(1, 3);
+            }
+
+            UnitQuaternion slerp(const UnitQuaternion& p, const double& t);
+            static inline UnitQuaternion Identity() {
+                return (arma::vec4({1.0, 0.0, 0.0, 0.0}));
+            }
+        };
+    }  // namespace geometry
```
Comment (TrentHouliston) [inline shared/utility/math/geometry/UnitQuaternion.hpp:146]: This shouldn't be here

**Review by TrentHouliston — CHANGES_REQUESTED** [review]
Found a misplaced comment!

File: shared/utility/math/geometry/UnitQuaternion.hpp (hunk 1 above)
Comment (KipHamiltons) [reply shared/utility/math/geometry/UnitQuaternion.hpp:146]: It actually should though
the way this was written, it has two namespaces within that one. We _could_ fully qualify each of them, but that seems less nice than the current situation.
The author did something similar in `Rotation3D`, as well as `Configuration.hpp`, `FileWatch.hpp` and `Script.hpp` :p

**Review by TrentHouliston — APPROVED** [review]
My bad I didn't see the opening

**Pull request merged**